lib/legacy/polymer.dom.html

```diff
@@ -235,6 +235,64 @@
     }
   }
 
+  /**
+   * @function
+   * @param {boolean=} deep
+   * @return {!Node}
+   */
+  DomApi.prototype.cloneNode;
+  /**
+   * @function
+   * @param {Node} node
+   * @return {!Node}
+   */
+  DomApi.prototype.appendChild;
+  /**
+   * @function
+   * @param {Node} newChild
+   * @param {Node} refChild
+   * @return {!Node}
+   */
+  DomApi.prototype.insertBefore;
+  /**
+   * @function
+   * @param {Node} node
+   * @return {!Node}
+   */
+  DomApi.prototype.removeChild;
+  /**
+   * @function
+   * @param {Node} oldChild
+   * @param {Node} newChild
+   * @return {!Node}
+   */
+  DomApi.prototype.replaceChild;
+  /**
+   * @function
+   * @param {string} name
+   * @param {string} value
+   * @return {void}
+   */
+  DomApi.prototype.setAttribute;
```

Comment:
FYI closure's types define `setAttribute` as taking `(string|number|boolean)`, whereas typescript's lib defines it as taking only `string`. Here i took the typescript version, but let me know if you disagree.